recipes/pydevtool/meta.yaml

```
@@ -0,0 +1,44 @@
+{% set name = "pydevtool" %}
+{% set version = "0.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pydevtool-{{ version }}.tar.gz
```

Having a `name` variable (kinda) makes sense in the example to help people populate their pypi URLs quickly and correctly, but it's a "fake" kind of variable, because it's not like it can be changed within this recipe. I'd suggest to remove it. 

```suggestion
{% set version = "0.2.0" %}

package:
  name: pydevtool
  version: {{ version }}

source:
  url: https://pypi.io/packages/source/p/pydevtool/pydevtool-{{ version }}.tar.gz
```

---

Why not. Note that this was generated using the recommended template generator. If this is not wanted anymore, it should be fixed there.

---

As I wrote, the template makes sense (+/-) so that newcomers don't have to figure out the PyPI links themselves; for a final recipe, it's useless though.

---

As I said, I used Grayskull to generate this file. There is nothing that would prevent this tool to resolve the link and just leave the version as a variable.